packages/core/src/lib/implementation/persist.ts

```diff
@@ -31,12 +31,10 @@ export async function persistReport(
   const outputDir = persist.outputDir;
   const filename = persist.filename;
   const format =
-    persist.format && persist.format.length !== 0 ? persist.format : ['stdout'];
-  let scoredReport;
-  if (format.includes('stdout')) {
-    scoredReport = scoreReport(report);
-    console.log(reportToStdout(scoredReport));
-  }
+    persist.format && persist.format.length !== 0 ? persist.format : [];
```

Comment:
I believe this is now equivalent:

```suggestion
    persist.format ?? [];
```

Reply:
Let's try it)